### DISCUSSION TITLE
GSA Discord Bot - Function to Privately DM Users

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#590] [Suggestions]
Like DynoBot, KarlBot, and many others, they include the function of being able to privately message a user things like greetings, notifications, general information, and so on. I am interested in incorporating the bot into a more automated punishment system that would allow for users to be messaged privately by the bot in the event of being kicked, banned, etc. Currently, the system allows for private in-game messaging, but players may likely miss out on the message before being kicked/banned, or may be incentivized to cause additional disruptions before finally being removed. 

It's not a priority now, but I'd like to request a task function for GSA to direct the Discord bot to message a connected Discord user when  the task executes, and possibly with other prompts such as making a purchase on the Web Shop, receiving a message in their inbox, etc.

**Comment by mvaessen** [comment]
Just posting here what I posted on Discord, for other folks:

> Do note though, many people have DM’s disabled for bots. It’s one of the reasons why gsa isn’t using discord dm’s much. It’s not a very reliable / issue-less way of communication

**Comment by mvaessen** [comment]
Sending a private message (DM) to Discord users is now available on the dashboard. 
<img width="959" height="295" alt="Screenshot 2026-02-25 at 17 48 43" src="https://github.com/user-attachments/assets/3839884f-2289-411f-bd07-350753b31a2c" />
